data/phi1348/abo013/__cts__.xml

```diff
@@ -1,11 +1,12 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi1348.abo013" xml:lang="lat" groupUrn="urn:cts:latinLit:phi1348">
-            <ti:title xml:lang="lat">Tiberius</ti:title>
+            <ti:title xml:lang="eng">Tiberius</ti:title>
```

Comment:
I think because it wasn't Divus Tiberius, with names like this I wasn't always certain. And in terms of Suetonius name (in the closed pull request) we had discussed leaving the full Suetonius Tranquillus in the headers (or I thought we had).